lib/loader/tests/index.js

```diff
@@ -23,7 +23,9 @@ function test(file) {
 
   // should export memory
   assert(exports.memory instanceof WebAssembly.Memory);
-  assert(typeof exports.memory.compare === "function");
+
+  // NOTE: Namespace exports have been removed in 0.20
+  // assert(typeof exports.memory.compare === "function");
```

Comment:
yay, obliterate namespace

Reply:
It's still an open question to me to what extent we'd want to support more complex exports. In particular, I could imagine that when a `class` is exported, and it is used as, say, the return type of an exported function, that a wrapper would be nice. Right now, the two options are either to a) copy field by field to a JS object or b) pass an internal reference. The latter could be extended with a wrapper, but while it would function like a normal object, would have some hidden cost depending on the situation. Right now I'm leaning to leaving this TBD and tackle once use cases arise.